src/util/importJSON.ts

```diff
@@ -130,7 +131,11 @@ export const importJSON = (state: State, thoughtsRanked: Path, blocks: Block[],
 
     // update contextIndexUpdates
     const contextEncoded = hashContext(rootContext)
-    const childrenUpdates = contextIndexUpdates[contextEncoded] ? contextIndexUpdates[contextEncoded].children : []
+
+    const childrenUpdates =
+      contextIndexUpdates[contextEncoded]?.children ||
+      contextIndex[contextEncoded]?.children || []
```

Comment:
@shresthabijay We can clean this up a bit with the [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining).